### DISCUSSION TITLE
API 403 forbidden issue (Projectwise Insight API - Get Project Reports)

### DISCUSSION THREAD

**Discussion by jaydenlzx** [iTwin/itwinjs-core#2153] [Q&A] [answered]
One of my vendor encountered a API 403 forbidden issue when they (jessicatanhc@optimus-pw.com)tried to access Projectwise Insight API – Get Project Reports. 

However, I can access the API with my optimus account (Jayden_lim@optimus-pw.com) but there isn’t any report inside (even though it should contain the insights reports). We tried to give them administrator rights in PWDM but they still couldn’t access. 

Is it because they don’t have admin rights in Projectwise Insights. We also realised that there isn’t any setting for access rights in Projectwise Insights. How can we resolve this issue?

**Comment by jaydenlzx** [comment, marked as answer]
![API 403](https://user-images.githubusercontent.com/80751784/131061835-0503f84b-ec0f-4001-8bda-e4e00b395b98.png)

**Reply by arnob-mallick** [reply]
Sorry for the confusion. The 'Insights API' you are trying to use here is not what you want. It is a separate thing entirely from Project Insights. The Insights API's intended use is for broader reporting needs and currently only supports iModels as a data source.

The Project Insights API / Documentation is accessible via our Power BI connectors. Please submit a ticket through richele.demesa@bentley.com to reach the Project Insights team for more details and assistance.
